### PR TITLE
add bid + bidder position documentation

### PR DESCRIPTION
**Pull request by erikdstock**
This PR adds documentation for bids and bidder positions to doppler. I think I hit everything, but:
- Left off the example section because it was getting a 403- perhaps doppler is not a trusted app + bids/positions are sensitive
- i noticed several other related endpoints [eg bidders] are not documented.
- rspec selenium tests wouldn't run at all. i'm not sure if this is an issue with my local setup or something else.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -330,5 +330,8 @@ DEPENDENCIES
   uglifier
   warden
 
+RUBY VERSION
```
Comment (dblock) [inline Gemfile.lock]: Seems iffy. This shouldn't be locked to 2.3.0 much less to a p0 one. Maybe move it up to 2.3.1?

**Comment by dblock** [comment]
Capybara tests and such should work. LMK if you need any help.

**Comment by erikdstock** [comment]
Will update, thanks.

File: Gemfile.lock (hunk 1 above)
Comment (erikdstock) [inline Gemfile.lock]: Yep- 2.3.0 was set in the Gemfile, so maybe this was a result of me adding it to my .ruby-version. Other than that I don't know what changed.

File: app/views/content/docs.md [hunk 2]
```diff
@@ -13,19 +13,34 @@
...
+* [Status](/docs/status)
 * [User Details](/docs/user_details)
-* [Collections](/docs/collections)
-* [Collection Items](/docs/collection_items)
-* [Images](/docs/images)
-* [Search](/docs/search)
+* [Users](/docs/users)
+
+<!--
+### Undocumented
```
Comment (erikdstock) [inline app/views/content/docs.md]: I left these in so they could be easily tracked and moved into place in the future. They could also be an issue in this repo.

**Comment by erikdstock** [comment]
cc @sweir27

**Comment by erikdstock** [comment]
resolves #116

File: app/views/content/docs.md [hunk 3]
```diff
@@ -13,19 +13,34 @@
...
+* [Fairs](/docs/fairs)
 * [Genes](/docs/genes)
-* [Profiles](/docs/profiles)
+* [Images](/docs/images)
 * [Partners](/docs/partners)
+* [Profiles](/docs/profiles)
+* [Sales](/docs/sales)
+* [Search](/docs/search)
 * [Shows](/docs/shows)
-* [Fairs](/docs/fairs)
```
Comment (sweir27) [inline app/views/content/docs.md]: 👏 yay alphabetization!

File: app/views/content/docs/bidder_positions.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+[Documentation](/docs) &gt; Resources &gt; Bidder Positions
+
+## Bidder Positions API
+
+A bidder creates [bidder positions](/docs/bidder_positions).
+
+``` alert[danger]
+The Bidder Positions API is restricted to trusted applications and admin users.
```
Comment (sweir27) [inline app/views/content/docs/bidder_positions.md]: #minor might want to be consistent about the language here and under `Bidders`

File: app/views/content/docs/bids.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+[Documentation](/docs) &gt; Resources &gt; Bids
+
+## Bids API
+
+An bid is a theoretical bid created while processing a [bidder position](/docs/bidder_positions) (following that Auction's rules regarding increments, reserve pricing and so on).
```
Comment (sweir27) [inline app/views/content/docs/bids.md]: #typo `"A bid ..."`

I'm not sure where the right place is, but we could link to @dblock's blog post about english auctions that explains this modeling http://artsy.github.io/blog/2014/04/17/building-an-english-auction-with-mongodb/

File: app/views/content/docs/sales.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+This endpoint accepts the following parameters.
+
+Name       | Value                                  |
+----------:|:---------------------------------------|
+live       | Boolean- filter live auctions only     |
+is_auction | Boolean- filter auctions only          |
+published  | Boolean- filter published auctions only|
+
+``` alert[danger]
+Only authorized users/applications may access **unpublished** sales (where published=false)
```
Comment (sweir27) [inline app/views/content/docs/sales.md]: This is a little misleading, as the API doesn't support passing `published=false` right now. The `unpublished` warning is more relevant when filtering by `live` or `is_auction`

**Comment by sweir27** [comment]
Few minor comments, otherwise this looks okay to me!

**Comment by erikdstock** [comment]
Since the Bidder swagger docs were missing but all other admin/trusted endpoints are in there I went ahead and added them: artsy/gravity#10787

File: app/views/content/docs/sales.md (hunk 6 above)
Comment (erikdstock) [inline app/views/content/docs/sales.md]: I don't fully understand what you mean here. What does the `published` param do, with and without `live` and `is_auction`?

File: app/views/content/docs/bidder_positions.md (hunk 4 above)
Comment (erikdstock) [inline app/views/content/docs/bidder_positions.md]: Addressed it across all endpoints

File: app/views/content/docs.md (hunk 2 above)
Comment (erikdstock) [inline app/views/content/docs.md]: Taking these out because they are in Doppler issues now.

**Comment by erikdstock** [comment]
Alright, the gravity PR above is merged and into production- good to merge @sweir27?

File: app/views/content/docs.md [hunk 7]
```diff
@@ -13,19 +13,32 @@
...
-* [Images](/docs/images)
-* [Search](/docs/search)
+* [Users](/docs/users)
+
+
+[comment]: # ( ### Undocumented )
+[comment]: # ( * [Collector Profiles](#) )
+[comment]: # ( * [Docs](#)               )
+[comment]: # ( * [Partner Contacts](#)   )
+[comment]: # ( * [Sale Artworks](#)      )
```
Comment (erikdstock) [inline app/views/content/docs.md]: I did research on SO to change these from HTML comments to markdown links that won't render at all, so no 
```
<!--
* [Missing Endpoints](/routes)
--> 
```
in the client's final markup.

File: app/views/content/docs/bidders.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+A bidder is a [user](/docs/users) registered for a specific [sale](/docs/sales).
+
+``` alert[danger]
+The Bidders API is restricted to authorized applications/users.
+```
+
+#### Retrieving Bidders
+
+Users can retrieve their own applications by following the [bidders](#{ArtsyAPI.artsy_api_root}/bidders) link from [root](#{ArtsyAPI.artsy_api_root}).
```
Comment (sweir27) [inline app/views/content/docs/bidders.md]: "Users can retrieve their own applications" doesn't seem right

File: app/views/content/docs/sales.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
+[Documentation](/docs) &gt; Resources &gt; Sales
+
+## Sales API
+
+Sales and Auctions are available at the sales endpoint.
+
+#### Retrieving Sales
+
+Users can retrieve their own applications by following the [sales](#{ArtsyAPI.artsy_api_root}/sales) link from [root](#{ArtsyAPI.artsy_api_root}).
```
Comment (sweir27) [inline app/views/content/docs/sales.md]: ditto

File: app/views/content/docs.md (hunk 7 above)
Comment (erikdstock) [inline app/views/content/docs.md]: No! Ok it is ticketed anyways

**Comment by sweir27** [comment]
Looks great!

**Pull request merged**